### ISSUE TITLE
requests twice for use

### ISSUE THREAD

**Issue by rdp** [vertiginous/pik#21]
Somewhat odd...

C:\dev\ruby\rubygems>pik 191
Select which Ruby you want:
1. 191: ruby 1.9.1p129 (2009-05-12 revision 23412) [i386-mingw32]
2. 191: ruby 1.9.1p243 (2009-07-16 revision 24175) [i386-mingw32]
?  2
Select which Ruby you want:
1. 191: ruby 1.9.1p129 (2009-05-12 revision 23412) [i386-mingw32]
2. 191: ruby 1.9.1p243 (2009-07-16 revision 24175) [i386-mingw32]
?  2
Using ruby 1.9.1p243 (2009-07-16 revision 24175) [i386-mingw32]

**Comment by vertiginous** [comment]
Is this with 0.2.2?  I thought I had this fixed.

Successfully installed pik-0.2.2
    1 gem installed

```
C:\scripts\repo\pik>pik_install C:\bin
Thank you for using pik.

Installing to C:\bin

cp C:/ruby/191-p243-mingw32/lib/ruby/gems/1.9.1/gems/pik-0.2.2/bin/../tools/pik.bat C:\bin
mkdir -p C:\bin/pik
cp C:/ruby/191-p243-mingw32/lib/ruby/gems/1.9.1/gems/pik-0.2.2/bin/../tools/pik/pik.exe C:\bin/pik

pik is installed

C:\scripts\repo\pik>pik 191
Select which Ruby you want:
1. 191: ruby 1.9.1p129 (2009-05-12 revision 23412) [i386-mingw32]
2. 191: ruby 1.9.1p243 (2009-07-16 revision 24175) [i386-mingw32]
?  2
```

**Comment by vertiginous** [comment]
Fixed somewhere around 0.1.1
